helm/prometheus/templates/rules.yaml

```diff
@@ -8,10 +8,11 @@ metadata:
     heritage: {{ .Release.Service }}
     prometheus: {{ .Values.prometheusLabelValue | default .Release.Name | quote }}
     release: {{ .Release.Name }}
+    role: alert-rules
```

Comment:
Could these be `recording-rules` as well as `alerting-rules` ?

Reply:
That is correct.

Reply:
Totally agree, that labels is used on all other configmap definitions of rules. Formally at some point this Chart should migrate to define proper PrometheusRules objects instead and at that point this labels can be "fixed".

Reply:
@richerve Would your suggestion be to simply remove the `role: alert-rules` label entirely at that point?

Reply:
Yes because at the end `PrometheusRule` is an explicit CRD so there will be no more "ConfigMap that serves as Prometheus rule definition"

Reply:
Thanks for that :-) Pr's been merged so we don't need to add this role any longer

Reply:
@vsliouniaev indeed the role label will not be needed but the name of the rule (now PrometheusRule) is the main purpose of these PR.